packages/ontario-frontend-cli/create/create.js

```diff
@@ -42,7 +42,8 @@ async function createNewProject(answers, options) {
     createDate: new Date().toISOString(),
     projectName: answers.projectName,
     projectDescription: answers.projectDescription,
-    ESLint: answers.ESLint,
+    esLint: answers.esLint,
+    prettier: answers.prettier,
```

Comment:
Can we make these boolean variables more clearly boolean?  Like `addESLint`, `addPrettier`?